plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -50,6 +57,21 @@ func getMaxVols(defaultVal int) int {
 	return defaultVal
 }
 
+// getHardPodAffinityWeight checks the hard pod affinity weight environment variable, otherwise returning a default value
```

Comment:
(I see you were following the pattern of the PD max volumes thing ... but that should probably have been a command-line argument too. No need to change that one, but if you don't mind, please make this a command-line argument.)